data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -67,7 +67,8 @@ t_europepmc_response_by_normalized_title AS (
 t_hypothesis_annotation_with_doi AS (
   SELECT
     *,
-    REGEXP_EXTRACT(annotation.uri, r'(10\.\d{3,}[^v]*)v?') AS source_doi
+    REGEXP_EXTRACT(annotation.uri, r'(10\.\d{3,}[^v]*)v?') AS source_doi,
+    REGEXP_EXTRACT(annotation.uri, r'10\.\d{3,}.*v([1-9])') source_version
```

Comment:
Should be `REGEXP_EXTRACT(annotation.uri, r'10\.\d{3,}.*v([1-9])') AS source_version`?